File: programmers/lv0/코딩기초트레이닝/출력과연산/홀짝구분하기/Solution.cpp
#include <iostream>

using namespace std;

int main(void) {
    int n;
    cin >> n;
//    if (n % 2 == 0) {
//        cout << n << " is even" << endl;
//    }
//    else {
//        cout << n << " is odd" << endl;
//    }
    cout << n << " is " << (n % 2 == 0 ? "even" : "odd") << endl;
    return 0;
}

File: programmers/lv0/코딩기초트레이닝/출력과연산/홀짝구분하기/Solution.go
package main

import "fmt"

func main() {
	var a int
	fmt.Scan(&a)
	isEven := a%2 == 0
	if isEven {
		fmt.Printf("%d is even\n", a)
	} else {
		fmt.Printf("%d is odd\n", a)
	}
}
